docs/source/usage.rst

```diff
@@ -142,9 +146,9 @@ the ``--reveal-prefix`` command line flag to point to the local copy.
 
 This will create file ``your_talk.slides.html``, which you should be able to 
 access with ``open your_talk.slides.html``. To access the speaker notes, press 
-``s`` after the slides load and they should open in a new window.
-
-This should also allow you to use your slides without an internet connection.
+``s`` after the slides load and they should open in a new window. Keep in mind
+that if you want a functional timer inside the speaker notes, you need to serve
```

Comment:
This is somewhat complicated syntax. I also think I’d prefer a caveat with no explanation of how to fix it to come earlier in the paragraph. The explanation would be given by reference with a hyperlink to a new section header that lives above the statement about the https server which should explicitly mention the bit about the timer.

Reply:
That's OK for me.